src/app/video-courses/components/video-courses-list/video-courses-list.component.ts

```diff
@@ -19,18 +19,26 @@ export class VideoCoursesListComponent implements OnInit, OnDestroy {
 
   private coursesChunkSize = 3;
 
-  constructor(
-    private videoCoursesService: VideoCoursesService,
-    private router: Router,
-    private route: ActivatedRoute,
-  ) {}
+  private searchSubject: Subject<string>;
```

Comment:
sometimes in the end of observable name people set $, like searchSubject$
https://angular.io/guide/rx-library#naming-conventions-for-observables